app/models/concerns/exportable_plan.rb

```diff
@@ -127,12 +132,25 @@ def prepare_coversheet
 
   # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
   def prepare_coversheet_for_csv(csv, _headings, hash)
+    csv << [_("Title: "), _("%{title}") % { title: title }]
     csv << [if hash[:attribution].many?
               _("Creators: ")
             else
               _("Creator:")
             end, _("%{authors}") % { authors: hash[:attribution].join(", ") }]
-    csv << ["Affiliation: ", _("%{affiliation}") % { affiliation: hash[:affiliation] }]
+    if hash[:investigation].present? 
+      csv <<  [_("Principal Investigator: "), _("%{investigation}") % { investigation: hash[:investigation].map(&:name).join(', ') }] 
+    end
+    if hash[:data_curation].present? 
+      csv << [_("Date Manager: "), _("%{data_curation}") % { data_curation: hash[:data_curation].map(&:name).join(', ') }] 
+    end
+    if hash[:pa].present? 
+      csv << [_("Project Administrator: "), _("%{pa}") % { pa: hash[:pa].map(&:name).join(', ') }] 
+    end
+    if hash[:other].present? 
+      csv << [_("Contributor: "), _("%{other}") % { other: hash[:other].map(&:name).join(', ') }] 
+    end
+    csv << [_("Affiliation: "), _("%{affiliation}") % { affiliation: hash[:affiliation] }]
```

Comment:
We should follow the syntax that is used in the rest of the code for checking present values, something like:
```ruby
csv << if hash[:investigation].present?
   [_("Principal Investigator: "), _("%{investigation}") % { investigation: hash[:investigation].map(&:name).join(', ') }]
end
```

Reply:
Just to clarify, won't this still trigger a write operation even if the hash[:investigation] is not present? I was thinking to follow line 163 or line 167's style since we want to skip this writing if the attribute is not available

Reply:
You are right.

Left as my proposal it would add a nil value if hash[:investigation] is not present, I think CSV would add a blank line if that was the case but I am not certain. Best leave it as is.